### DISCUSSION TITLE
React hosting with custom domain gives 404 for other projects

### DISCUSSION THREAD

**Discussion by devashish2024** [community/community#127802] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have my portfolio at [ashishagarwal.is-a.dev](https://ashishagarwal.is-a.dev). It's also set as my github account's main link (therefore, ashishagarwal2023.github.io will go to ashishagarwal.is-a.dev and so for all subroutes).

My root (ashishagarwal2023.github.io repo or my portfolio) is built with next.js (react). Now, if I go to [ashishagarwal2023.github.io/chatgpt-clone](https://ashishagarwal2023.github.io/chatgpt-clone), it will redirect me to [ashishagarwal.is-a.dev/chatgpt-clone](https://ashishagarwal.is-a.dev/chatgpt-clone) but the next.js gives 404.

Any way to fix it? All my repos are being redirected and acheiving 404.

**Comment by Sicilianii** [comment]
Had the same problem, here's the link
https://stackoverflow.com/questions/11577147/how-to-fix-http-404-on-github-pages

If you find this as helpful, please mark this comment as an answer :)
